src/layouts/Default.vue

```diff
@@ -0,0 +1,47 @@
+<script>
+import FooterLayout from "./Footer.vue";
+import HeaderLayout from "./Header.vue";
+
+export default {
+  components: { FooterLayout, HeaderLayout },
+  name: "DefaultLayout",
+};
+</script>
+
+<template>
+  <div class="layout">
+    <div class="stripe" />
+    <HeaderLayout />
+    <main>
+      <slot />
+    </main>
+    <FooterLayout />
+    <div class="stripe" />
+    <!-- Place this tag in your head or just before your close body tag. -->
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
```

Comment:
@alexmarple - This came in via a merge conflict from main. It was in `Head.html`, which we don't have in this branch. Feel free to relocate or delete as appropriate.